packages/store/src/internal/dispatcher.ts

```diff
@@ -33,10 +33,8 @@ export class InternalDispatcher {
   /**
    * Dispatches event(s).
    */
-  dispatch(actionOrActions: any | any[]): Observable<any> {
-    const result = this._ngxsExecutionStrategy.enter(() =>
-      this.dispatchByEvents(actionOrActions)
-    );
+  dispatch<T = any>(actions: ActionType | ActionType[]): Observable<T> {
```

Comment:
In v4 this will return `Observable<void>`, so we should not add any typing here.
See this item in the listed breaking changes for v4 (ref #827):
> The value passed in the observable returned from a dispatch call should be void, matching its interface

Reply:
I can add todo